EmployeeAdmin/src/com/controller/QAEmployeeController.java

```diff
@@ -0,0 +1,111 @@
+package com.controller;
+
+import java.util.List;
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+
+import org.codehaus.jackson.JsonGenerationException;
+import org.codehaus.jackson.map.JsonMappingException;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.codehaus.jettison.json.JSONException;
+import org.codehaus.jettison.json.JSONObject;
+
+import com.employee.QAEmployee;
+
+/**
+ * This class is used to make actions on QAEmployee:
+ * GET - view all the QAEmployees + filter QAEmployees
+ * POST - add new QAEmployees
+ * PUT - update existing QAEmployee
+ */
+@Path("/employeeList")
+public class QAEmployeeController {
+	
+	private ObjectMapper mapper = new ObjectMapper();
+
+	static List<QAEmployee> employeeList = new ArrayList<QAEmployee>(){
+	    {
+	        add(new QAEmployee(1, "Mada", "mada@gmail.com", 23, true));
+	        add(new QAEmployee(2, "Ana", "ana@gmail.com", 21, false));
+	        add(new QAEmployee(3, "Ema", "ema@gmail.com", 33, true));  
+	    }
+	 };
+	
+	
+	@GET
+	@Produces(MediaType.APPLICATION_JSON)
+	public String ShowEmployeeList(
+			@DefaultValue("id") @QueryParam("filter") String filter, 
```

Comment:
as a best practice, you could have a constants file for 'filter', 'value', etc. This is a really small project and you don't need it, but in a larger project, you should have it. If I'm not wrong, you have used these in at least 2 places, (if the project have been bigger, probably more).